build.sbt

```diff
@@ -1,34 +1,21 @@
 import ReleaseTransformations._
-lazy val sparkVersion = "3.2.0"
+
+lazy val sparkVersion = "3.5.0"
 
 ThisBuild / sonatypeCredentialHost := "s01.oss.sonatype.org"
 ThisBuild / sonatypeRepository := "https://s01.oss.sonatype.org/service/local"
 
 lazy val root = (project in file("."))
   .configs(IntegrationTest)
   .settings(
-    name                 := "spark-pinecone",
-    organizationName     := "Pinecone Systems",
+    name := "spark-pinecone",
+    organizationName := "Pinecone Systems",
     organizationHomepage := Some(url("http://pinecone.io/")),
-    organization         := "io.pinecone",
+    organization := "io.pinecone",
     licenses := Seq(("Apache-2.0", url("http://www.apache.org/licenses/LICENSE-2.0"))),
     description := "A spark connector for the Pinecone Vector Database",
-    developers := List(
```

Comment:
**question:** Should we maybe keep the developers list and add you to it?

Reply:
I think it adds noise to the sbt and the contributors for each release can be found in the changelogs or the repository's main page.

Reply:
Plus other than the name, it's a lot of redundant information